includes/syslog.php

```diff
@@ -166,6 +173,12 @@ function process_syslog($entry, $update)
 
         unset($os);
     }//end if
+    elseif (true == Config::get('syslog_debug')) {
+        // For finding unknowen Hosts
+        $fd = fopen('/opt/librenms/logs/Debug.log', 'a');
```

Comment:
don't open a new file , just `Log::<level>()`

Reply:
I tryed to use 
Log::debug("Can't process host: " . $entry['host']);
but I can not see the outpout in the Debug File.

Reply:
It goes to the standard `logs/librenms.log` then.